images/win/scripts/SoftwareReport/SoftwareReport.Browsers.psm1

```diff
@@ -52,4 +53,26 @@ function Get-SeleniumWebDriverVersion {
     $versionFileName = "versioninfo.txt";
 	$webDriverVersion = Get-Content -Path "$driverPath\$versionFileName"
     return "$driverName $webDriverVersion"
-}
\ No newline at end of file
+}
+
+function Build-BrowserWebdriversEnvironmentTable {
+    return @(
+        @{
+            "Name" = "CHROMEWEBDRIVER"
+            "Value" = $env:CHROMEWEBDRIVER
+        },
+        @{
+            "Name" = "EDGEWEBDRIVER"
+            "Value" = $env:EDGEWEBDRIVER
+        },
+        @{
+            "Name" = "GECKOWEBDRIVER"
+            "Value" = $env:GECKOWEBDRIVER
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-Path-With-Link($_.Value)
```

Comment:
I guess we don't need to use the function here. These folders are not symlinked.
It makes sense to put symlinks only for Android variables